test/integration-tests/testset_sdk/javascript/functionProvider.js

```diff
@@ -1,28 +1,31 @@
+// TODO: We need to fix this Jochen
```

Comment:
What has to be fixed?
The idea here is to use the actual repo as default. We could make this configurable from outside.

Reply:
Toms comment. Just left it

Reply:
This has been already addressed, so it could be removed.